tests/test_discretisation/test_increasing_width_discretiser.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import pytest
+from sklearn.exceptions import NotFittedError
+
+from feature_engine.discretisation import IncreasingWidthDiscretiser
+
+
+def test_automatically_find_variables_and_return_as_numeric(df_normal_dist):
```

Comment:
This test structure is inheritance from my lesser programming skills from when I started :p

I think it would be better to break down this test into smaller tests:

- Test init params work and raise error when non permitted
- test fit method
- test transform method

Would you be able to do that? 

And did you run coverage on this class? it tells you which bits of code are not being tested yet.

Thank you!